packages/@expo/cli/bin/cli.ts

```diff
@@ -83,6 +84,8 @@ if (!isSubcommand && args['--help']) {
     prebuild,
     'run:ios': runIos,
     'run:android': runAndroid,
+    // Don't document this command
```

Comment:
explain why it's not documented somewhere

Reply:
I don't want people to use this command because it's a temporary workaround until we can figure out how to use `npx expo export` to generate and embed an update that's identically formatted to what we upload to EAS Update.

Reply:
I get that it's internal, just saying to explain to people working on this codebase why it's not documented. I also meant to suggest communicating the internal-only aspect of this command in its name, like export:embed_INTERNAL_ONLY. Anyone who types in that command knows not to rely on it.